api.go

```diff
@@ -373,15 +373,30 @@ func (api *API) delSilence(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *API) listSilences(w http.ResponseWriter, r *http.Request) {
-	sils, err := api.silences.All()
+	var (
+		num    = r.FormValue("n")
```

Comment:
I couldn't think of a good short value for `offset`, but `n` seemed comfortable.

The original idea was to present `page` to the user in the URL, and to use `offset` in the API. Maybe it's confusing, but it made sense to me initially to view pages starting at 1, but using a zero based offset in the api.

- happy to change `n` to limit
- where do you stand on starting at 0 or 1? I don't really like showing `page=0` to the user, but I would prefer that to sending `page=1` to the API and then having to subtract one there for pulling IDs from the cached ID slice.